test/one/profiler/test/TestProcess.java

```diff
@@ -93,6 +93,19 @@ public TestProcess(Test test, Os currentOs, String logDir) throws Exception {
             // Give the JVM some time to initialize
             Thread.sleep(700);
         }
+
+        if (test.cpu() > 0) {
+            List<String> pinCpuCmd = new ArrayList<>();
+            pinCpuCmd.add("taskset");
+            pinCpuCmd.add("-acp");
+            pinCpuCmd.add(String.valueOf(test.cpu()));
+            pinCpuCmd.add(String.valueOf(p.pid()));
```

Comment:
Since the number of parameters is static here, you can simplify this to 
```
String[] tasksetCmd = {"taskset", "-acp", ...};
```
Or even simpler, call `Runtime.exec`.